deployment/model-testing/check_model_dependencies.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Validate a Model's Environment Contains the Required Dependencies\n",
+    "This example demonstrates how to check that a model's environment contains all the dependencies it needs to run.\n",
+    "This allows developers and data scientists to validate the environment locally before attempting to deploy the\n",
+    "model to Verta."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Imports"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "from verta import Client\n",
+    "from verta.environment import Python\n",
+    "from verta.registry import VertaModelBase, verify_io, _check_model_dependencies as check_model_dependencies\n",
+    "from verta.registry.entities import RegisteredModel,  RegisteredModelVersion\n",
+    "from verta.utils import ModelAPI"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Verta Client Setup"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Use local env vars or uncomment and fill out the lines below:\n",
+    "# os.environ['VERTA_EMAIL'] = ''\n",
+    "# os.environ['VERTA_DEV_KEY'] = ''\n",
+    "# os.environ['VERTA_HOST'] = ''\n",
+    "\n",
+    "client = Client()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Example Model Class"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "class ExampleModel(VertaModelBase):\n",
+    "    def __init__(self, artifacts):\n",
+    "        pass\n",
+    "\n",
+    "    @verify_io\n",
+    "    def predict(self, input: pd.DataFrame) -> pd.DataFrame:\n",
+    "        return input.iloc[:1]"
```

Comment:
`predict()` in deployed models doesn't support taking `DataFrame` objects. `input` will always be some JSON-serializable object, like a `list`.

Reply:
fixed